Update query-account-audit-logs-nrauditevent.mdx (#20444)
newrelic/docs-website: src/content/docs/accounts/accounts/account-maintenance/query-account-audit-logs-nrauditevent.mdx

    ```
  </Collapser>

<Collapser
    id="user-login"
    title="What users have logged into our organization?"
  >

>     To query for information about users who have logged into your organization, you can use something like:

Reviewer: ```suggestion
    To query users who logged into your organization, use a query similar to the following:
```
